src/components/Common/TitleAndList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import remcalc from 'remcalc'
+import ReactMarkdown from 'react-markdown'
+import styled from 'styled-components'
+import { Col, Row } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import { Subtitle, BodyPrimary, SectionTitle } from '../Typography'
+
+const OuterPaddings = ({ children }) => (
+  <Padding vertical={{ smallPhone: 3, tablet: 4 }}>{children}</Padding>
+)
+
+const ParagraphWithOptionalPadding = styled(BodyPrimary)`
+  padding-bottom: ${remcalc(24)};
+  &:last-child {
+    padding: 0;
+  }
+`
+const ItemTitle = ({ children, ...props }) => (
+  <Subtitle as="h3" noPadding bold {...props}>
+    {children}
+  </Subtitle>
+)
+
+const TitleAndList = ({ title, list, bg = 'white', extraContent }) => {
+  return (
+    <OuterPaddings>
+      <Row>
+        <Col width={[1, 1, 1, 6 / 12]}>
+          <Padding bottom={3}>
+            <SectionTitle small noPadding reverse={bg === 'dark'}>
+              {title}
+            </SectionTitle>
+          </Padding>
+        </Col>
+        <Col width={[1, 1, 1, 6 / 12]}>
+          {typeof list === 'string' ? (
+            <ReactMarkdown
+              renderers={{
+                heading: props => {
+                  const Comp = <ItemTitle reverse={bg === 'dark'} {...props} /> // eslint-disable-line react/no-display-name
+                  return Comp
+                },
+                paragraph: props => {
+                  const Comp = (
+                    <ParagraphWithOptionalPadding
+                      noPaddingTop
+                      muted={bg === 'dark'}
+                      reverse={bg === 'dark'}
+                      {...props}
+                    />
+                  ) // eslint-disable-line react/no-display-name
+
+                  return Comp
+                }
+              }}
+              source={list}
+            />
+          ) : null}
+          {Array.isArray(list)
+            ? list.map((el, idx) => (
+                <React.Fragment key={idx}>
+                  <ItemTitle reverse={bg === 'dark'}>{el.title}</ItemTitle>
+                  <ParagraphWithOptionalPadding
+                    noPaddingTop
+                    muted={bg === 'dark'}
+                    reverse={bg === 'dark'}
+                  >
+                    {el.body}
+                  </ParagraphWithOptionalPadding>
+                </React.Fragment>
+              ))
+            : null}
+          {extraContent}
```

Comment:
I don't totally agree with this...
Yes, currently there's not a scenario where a something else than a `StylisedLink` is used. 

However, 'forcing' it to be a `StylisedLink` and showing it or not with props,  causes some problems :
- the "natural" problem of not being able to add something else (if needed in the future) without having to refactor existing usages of this component
- it's not obvious for the developer what to pass in case there's the need of a stylised link. Is it the `to` prop? Is it the label? I should at least have to have a `to` and a `label` prop, right? 
- what if we want to provide an event handler (`onClick` for example)? Should that be a prop? Should we cascade the rest of the `TitleAndList` props to the `StylisedLink`? There are too many possibilities and that causes confusion to the developer using this.

This a common pattern in React, that avoids overloading our component with props that may or not be neded, and gives flexibility to the "user".